molecule/testinfra/common/test_grsecurity.py

```diff
@@ -81,7 +81,6 @@ def test_grsecurity_kernel_is_running(host):
     [
         ("kernel.grsecurity.grsec_lock", 1),
         ("kernel.grsecurity.rwxmap_logging", 0),
-        ("vm.heap_stack_gap", 1048576),
```

Comment:
(repeat) IMO keeping the testinfra checks makes sense, in case the metapackage-set values get borked somehow.